indico/modules/events/static/templates/download_notification_email.txt

```diff
@@ -0,0 +1,16 @@
+{% extends 'emails/base.txt' %}
+
+{% block subject %}Offline copy of an event ready for download{% endblock %}
+{% block footer_title %}Offline copy of the event "{{ event_title }}" ready for download.{% endblock %}
+
+{% block header_recipient -%}
+    {{ user.full_name }}
+{%- endblock %}
+
+{% block body -%}
+The offline copy of the event "{{ event_title }}" is ready for download.
+
+Download link: {{ link }}
+
+Best Regards,
```

Comment:
We already have a standard footer and "best regards" is not a good fit in an email sent by a machine.